### PR TITLE
test issue 31

### PR DESCRIPTION
**Pull request by ocefpaf**
@grahamanderson and @jorisvandenbossche this PR adds a test for issue #31. I cannot reproduce the issue on Windows and Linux. Let's keep an eye for Travis-CI to see if we can reproduce it on OS X.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Travis-CI is passing on my instance, see https://travis-ci.org/ocefpaf/geopandas-feedstock/builds/313583866

@grahamanderson and @jorisvandenbossche, without access to a Mac it is hard for me to do anything else to debug this :unamused:

**Comment by jorisvandenbossche** [comment]
Thanks for checking here. I also don't have a Mac, so can't help out here ..

**Comment by grahamanderson** [comment]
@ocefpaf is there any way you can check (remotely) on my Mac? And, is it possible  to include the Fiona devs—which is probably where the problem lies?
Macs make up a sizable percentage of the data science’y folks. On my end, I’ll try a fresh install with pip today and report back. Thanks for giving us Mac users some love.

File: recipe/run_test.py [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+)
+
+# Read geopackage.
+gpkg = geopandas.read_file('geopackage.gpkg')
+
+
+# Compare.
+from pandas.testing import assert_frame_equal
+
+assert_frame_equal(gdf, gpkg)
```
Comment (ocefpaf) [inline recipe/run_test.py]: Test https://github.com/conda-forge/fiona-feedstock/issues/53

**Comment by ocefpaf** [comment]
> @ocefpaf is there any way you can check (remotely) on my Mac?

My guess is that latest OS X system are shipping with a bundled `geos`. Not sure though, I am guessing that b/c people are successfully using our conda packages in older system. Can you try to confirm that?

> And, is it possible to include the Fiona devs—which is probably where the problem lies?

In theory, a patch that I contributed, should fix that. So in a way the dev that should know about that me 😬 

> Macs make up a sizable percentage of the data science’y folks. On my end, I’ll try a fresh install with pip today and report back.

Indeed. We need a better way to build and test those packages for OS X.

BTW, can you try the packages in `defaults`, without `conda-forge`, just curious if they suffer from the same bug.

**Pull request closed without merging**